pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -3,9 +3,9 @@
 
 <head>
     <%- include('../partials/head'); %>
-    <link href="/stylesheets/theme.bootstrap.css" rel="stylesheet" />
-    <script src="/javascripts/jquery.tablesorter.min.js"></script>
-    <script src="/javascripts/jquery.tablesorter.widgets.js"></script>
+        <link href="/stylesheets/theme.bootstrap.css" rel="stylesheet" />
+        <script src="/javascripts/jquery.tablesorter.min.js"></script>
+        <script src="/javascripts/jquery.tablesorter.widgets.js"></script>
```

Comment:
It looks like your editor might have made some changes to the whitespace here. Please undo all of these (and the similar whitespace changes below), so that you only make changes that are necessary.